model/task_lifecycle.go

```diff
@@ -553,8 +556,15 @@ func doStepback(ctx context.Context, t *task.Task) error {
 		return nil
 	}
 
+	s := task.StepbackInformation{
+		StepbackDepth:             t.StepbackDepth + 1,
+		LastFailingStepbackTaskId: "",
```

Comment:
The default value for strings is the empty string so you don't need to define it here. Is it correct though that we don't want to be setting these values? Do you intend to start updating these in a future PR?

Reply:
Yes. For now, this ticket just adds the fields to the database to separate the logic of setting them appropriately and this.